.jshintrc

```diff
@@ -0,0 +1,17 @@
+{
+  "browser": false,
+  "node": true,
+  "globals": {
+    "require": true
+  },
+  "predef": {
+  },
+  "eqeqeq": true,
+  "quotmark": true,
+  "smarttabs": true,
+  "trailing": true,
```

Comment:
Smart tabs: http://jshint.com/docs/faq/

Trailing: looks like this was removed with v2.5, it warned for trailing
whitespace on lines

On Sunday, February 1, 2015, Raminder Singh <notifications@github.com>
wrote:

> In .jshintrc
> <https://github.com/qiao/PathFinding.js/pull/88#discussion_r23895774>:
>
> > @@ -0,0 +1,17 @@
> > +{
> > +  "browser": false,
> > +  "node": true,
> > +  "globals": {
> > +    "require": true
> > +  },
> > +  "predef": {
> > +  },
> > +  "eqeqeq": true,
> > +  "quotmark": true,
> > +  "smarttabs": true,
> > +  "trailing": true,
>
> What do smarttabs and trailing options do? I couldn't find them in the
> jshint docs.
>
> --
> Reply to this email directly or view it on GitHub
> <https://github.com/qiao/PathFinding.js/pull/88/files#r23895774>.
>